cfgov/agreements/jinja2/agreements/index.html

```diff
@@ -1,6 +1,8 @@
 {% extends "agreements/base_agreements.html" %}
 
-{% block title %}Credit card agreement database{% endblock %}
+{% block title %}
+    Credit card agreement database | Consumer Financial Protection Bureau
+{% endblock %}
```

Comment:
This is pretty trivial, but could we collapse these new newlines wherever they appear in this PR, for consistency (see, e.g., `cfgov/diversity_inclusion/jinja2/diversity_inclusion/form-submitted.html` a couple files down in this diff already had them)

```suggestion
{% block title -%}
    Credit card agreement database | Consumer Financial Protection Bureau
{%- endblock %}
```

Reply:
Done! https://github.com/cfpb/consumerfinance.gov/pull/6339/commits/7358f3030aa64a945a16baa6127f58011eec8680